docs/guide/http_cache.md

```diff
@@ -326,6 +336,8 @@ fastcgi_param SYMFONY_TRUSTED_PROXIES "193.22.44.22";
 
 Secondly we need to tell eZ Platform to change to use http based purge client *(specifically FosHttpCache Varnish purge client is used)*, and specify url Varnish can be reached on:
 
+Since eZ Platform 2.x, this following configuration is not required as eZ will read the Environment Variables set above.
```

Comment:
Can maybe mark this section as optional. More relevant as example if you need to set several purge servers for instance _(when you have for instance two varnish servers, typically one active and one passive)_.